### PR TITLE
[goreleaser] Support m1 mac

### LINKED ISSUE #58: Homebrew Tap does not support m1
### Installation Process
 
N/A

### Nova Version

N/A

### Expected Behavior

Installation should work on m1 mac machines

### Actual Behavior

https://github.com/FairwindsOps/pluto/issues/197

### Steps to Reproduce

1. Have an m1 mac
2. `brew install FairwindsOps/tap/nova`

### Additional Context

N/A

### References

- https://github.com/FairwindsOps/pluto/issues/197

I can submit a PR to fix this. I think we just need to add a goreleaser build for ARM + Darwin

### PR DESCRIPTION
**Pull request by lucasreed**
Fixes #58
Fixes https://github.com/FairwindsOps/pluto/issues/197

### CODE REVIEW

**Review by sudermanjr — APPROVED** [review]

**Comment by lucasreed** [comment]
Hold on merging this until I make sure it's working.

**Comment by lucasreed** [comment]
I think this should be good to go once a CODEOWNER approves.

**Review by rbren — APPROVED** [review]

**Pull request merged**
